package.json

```diff
@@ -41,7 +41,7 @@
     "command-line-args": "^5.0.2",
     "cron": "^1.4.1",
     "dotenv": "^5.0.1",
-    "eslint": "^4.19.1",
+    "eslint": "^5.15.0",
```

Comment:
We just need `eslint` in `devDependencies` only. When we do `npm i` in CircleCI, we automatically include the dev dependencies also. To just install just `dependencies` and not `devDependenies`, we need to run `npm i --only=production` - which we do not currently practice.